### ISSUE TITLE
Order of loaded models

### ISSUE THREAD

**Issue by renedominik** [arose/mdsrv#55]
When loading multiple structures, the order of their list in the menu is random.

**Comment by j0kaso** [comment]
yes. as far as I know, sorting is not possible. you could maybe define the order by loading each one after another with some waiting time in between.
